### PR TITLE
!!![v6r15] Logging and error reporting

### PR DESCRIPTION
**Pull request by fstagni**
I also changed the name of "Core.Security.File" to "Core.Security.ProxyFile" ("File" is really too of a generic name) and updated the references.

And some more minor fixes.

This PR is not complete: I prepared some security utilities modules to use DError, but what's today merged in v6r15 is not the completed version. So it will need anyway to be updated (I won't be at CERN for 2 weeks, so I push it now anyway...)

### CODE REVIEW

**Comment by chaen** [comment]
The change of class name should be advertised in the wiki so that projects can adapt their code

File: Core/Security/X509CRL.py [hunk 1]
```diff
@@ -47,7 +45,9 @@ def loadChainFromFile( self, crlLocation ):
       pemData = fd.read()
       fd.close()
     except Exception as e:
-      return S_ERROR( "Can't open %s file: %s" % ( crlLocation, str( e ) ) )
+      return DError(DErrno., "%s: %s" % ( crlLocation, str( e ) ) )
```
Comment (chaen) [inline Core/Security/X509CRL.py]: superfluous "str"

**Comment by fstagni** [comment]
Done the twiki, and updated the code.

**Comment by atsareg** [comment]
Please, confirm that this PR is finalized

**Comment by chaen** [comment]
No it is not. But Federico is absent for the moment. Do not merge please

**Comment by atsareg** [comment]
Reminder that this PR has conflicts that must be resolved

**Pull request closed without merging**